stormlog/_wandb/tracking.py

```
@@ -0,0 +1,471 @@
+                "rank": event_value(event, "rank"),
+            }
+        )
+
+    return sample_timeline_rows(timeline_rows)
+
+
+def sample_timeline_rows(rows: Sequence[dict[str, Any]]) -> list[dict[str, Any]]:
+    if len(rows) <= _TIMELINE_MAX_POINTS:
+        return list(rows)
+
+    stride = int(math.ceil(len(rows) / _TIMELINE_MAX_POINTS))
```

Ruff (0.15.10)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)